app/bundles/LeadBundle/Entity/LeadFieldRepository.php

```diff
@@ -14,6 +14,11 @@
  */
 class LeadFieldRepository extends CommonRepository
 {
+    /**
+     * @var array<int|string, array<string,mixed>>|null
+     */
+    private static ?array $fields;
```

Comment:
```suggestion
    private ?array $fields = null;
```
This is where the error comes from. Can we use just a private property instead of static? It generates hard to find bugs. For example in tests when the array is filled in one test and it remains the same for the rest of the tests no matter if the fields change or not.